cpu/stm32_common/periph/flashpage.c

```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @ingroup     cpu_stm32f0
+ * @ingroup     cpu_stm32_common
```

Comment:
I still think its not needed: if you compile it for one of the latter CPUs, you will get an compile error anyway, as `FLASHPAGE_SIZE` and `FLASHPAGE_NUMOF` are not defined. And if one is newly defining those values, the `flashpage` driver has to be tested in any case...